### DISCUSSION TITLE
Save

### DISCUSSION THREAD

**Discussion by Dostonlv** [zed-industries/zed#6472] [Feature Requests]
<img width="171" alt="Screenshot 2023-02-26 at 23 56 40" src="https://user-images.githubusercontent.com/97626504/221430932-7df2b09c-2312-4bf4-bc86-2905ce523a99.png">

If it is like this, how to save, I can't even save auto save from settings, what is the short cut for this

**Comment by widersky** [comment]
Ummm, I'm not sure I've understood your question correctly, but... `cmd + s`?

**Comment by JosephTLyons** [comment]
Hey @Dostonlv, are you having issues with save not working at all?  If you use `cmd-s`, does it not save your file?

**Comment by ghost** [comment]
Command + S or File -> Save

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
